docs/developer/api-reference/objects/stock-settings.mdx

```diff
@@ -0,0 +1,23 @@
+---
+id: stock-settings
+title: StockSettings
+hide_table_of_contents: false
+---
+
+Represents the channel stock settings.
+
+Added in Saleor 3.7.
+
+Note: this API is currently in Feature Preview and can be subject to changes at later point.
+
+```graphql
+type StockSettings {
+  allocationStrategy: AllocationStrategyEnum!
+}
+```
+
+### Fields
+
+#### [`allocationStrategy`](#) ([`AllocationStrategyEnum!`](../../../developer/api-reference/enums/allocation-strategy-enum))
+
+Allocation strategy options. Strategy defines the preference of warehouses for allocations and reservations.
```

Comment:
Could be simplified:
`Allocation strategy defines the preference of warehouses for allocations and reservations.`

Reply:
This is autogenerating from our schema so I need to update the schema on the feature branch first.